package.json

```diff
@@ -54,6 +54,7 @@
     "husky:install": "husky install",
     "test": "jest --maxWorkers=4",
     "test:integration": "jest --config jest.integration.config.ts --runInBand",
+    "test:e2e": "jest --config jest.e2e.config.ts",
```

Comment:
We're not using too many abbreviations normally, so I would not mind calling stuff `endToEnd` instead of `e2e`.

Reply:
done changed to end2end